includes/payment-methods/class-cc-payment-method.php

```diff
@@ -24,6 +24,7 @@ public function __construct( $token_service ) {
 		$this->stripe_id   = 'card';
 		$this->title       = 'Credit card / debit card';
 		$this->is_reusable = true;
+		$this->currencies  = [];
```

Comment:
I know that having an empty list of supported currencies to indicate that all currencies are supported is probably a little non-intuitive, so let me know if you have any other thoughts for this case if it all sounds a bit too crazy.

Reply:
I'd left a comment over [here](https://github.com/Automattic/woocommerce-payments/blob/9b55ee94f9e7e1fca222cff287293ba06beda529/includes/payment-methods/class-upe-payment-method.php#L55), but I've just added another in the CC payment method as well for completeness.